src/interactive/scala/tools/nsc/interactive/REPL.scala

```diff
@@ -104,29 +104,34 @@ object REPL {
     def doTypeAt(pos: Position): Unit = {
       comp.askTypeAt(pos, typeatResult)
       show(typeatResult)
+      ()
     }
 
     def doComplete(pos: Position): Unit = {
       comp.askTypeCompletion(pos, completeResult)
       show(completeResult)
+      ()
     }
 
     def doStructure(file: String): Unit = {
       comp.askParsedEntered(toSourceFile(file), keepLoaded = false, structureResult)
       show(structureResult)
+      ()
```

Comment:
I think I preferred `: Unit` but I could also go for `;()` winking monkey face.

It's a shame to increase vertical space.

Reply:
Actually, the problem with the idiom is that it silences one warning, but the problem remains, to be discovered later by `-Wnonunit-statement`.

Reply:
This is an example where the fault lies with the signature, as it should be `def show(): Unit`. Previously, it was `Option[Unit]`. Possibly, more inferred types should warn.